### ISSUE TITLE
[Splitter] Improve word dataset creation / balancing

### ISSUE THREAD

**Issue by felixdittrich92** [felixdittrich92/docTR-Synth-Generator#6]
Training set we generate at a minimum 1 sample for each word in the wordlist the maximum should be not limited

Validation set should be calculated by the number of training samples and a given percentage additional we need to ensure that at least each char is covered in the validation set
